src/PointerBox.tsx

```diff
@@ -1,30 +1,45 @@
 import React from 'react'
-import BorderBox, {BorderBoxProps} from './BorderBox'
+import Box, {BoxProps} from './Box'
 import Caret, {CaretProps} from './Caret'
+import {SxProp} from './sx'
+
+// FIXME: Make this work with BetterStyledSystem types
+type MutatedSxProps = {
+  sx?: {
+    bg?: string
+    backgroundColor?: string
+    borderColor?: string
+  }
+} & SxProp
```

Comment:
wow i wonder if this can be removed?

Reply:
🤷‍♂️  any ideas? I can't seem to coerce the types to match with sx without doing this. Couldn't find any other similar patterns either, so stuck a FIXME on it for now to come back to later. Would appreciate any suggestions 🙏